lib/numbers.md

```diff
@@ -5,20 +5,23 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What are the different kinds of numbers in Ruby?
 
-Replace me with your response...
+Ruby has 'Fixnum' numbers, or 'intigers' that are whole numbers, and a 'float' wich a non whole number with a decimel point.
 
 # What are some common operations and comparisons you would perform on numbers?
 
-Replace me with your response...
+I suppose the most common operation with numbers would be in a mathmatical equation to complete a code.  It will be necessary to find out what type of 
+number is being input by a user, and requesting that number be how the program needs it to be, like an integer or an even number
 
 # What is the difference between the `+` operation on a number versus on a String?
 
-Replace me with your response...
+'+' when used with a string will help ruby to assign a value.  When its used with a number, it would simply add, or complete the mathmatical
+equation.
 
 # If you have a _String_ `"20"` and want to perform a mathematical operation (like division or multiplication) on it, will it work? If yes, why? If not, how would you make it work?
 
-Replace me with your response...
+No, it will not work becuase Ruby sees numbers in "" as the character, instead of the value.
+If you but _String_ = 20 taking it out the the "" made it a value able to be used in a mathmatical operation
```

Comment:
We already gave the variable x the value of "Twenty" in the first line
X="20" (as ruby sees this as a string, not a numerical value that can be calculated with other numbers.
If we also defined X as the number '20' in a line right under that, it would be defined as the number as well.
x = '20'
x = 20
quotient = x / 4
puts "20 divided by 4 is " + quotient